### ISSUE TITLE
Add Xml property to ArrayModel class

### ISSUE THREAD

**Issue by rbsilva** [swagger-api/swagger-core#2669]
| Q                                 | A
| --------------------------------- | -------
| Bug or feature request?           |  feature
| Which Swagger-Core version?       | 1.5.19
| Which Java version?               | Java 8
| Which JAX-RS framework & version? | -

Actual behaviour:

When creating a ArrayModel:
```
new ArrayModel().items(....);
```
The output json is:
```
{
   "type":"array",
   "items":{"$ref":"#/definitions/Payment"}
}
```

Expected behaviour:

When creating a ArrayModel with Xml:
```
new ArrayModel().items(....).xml(new Xml().wrapped(true).name("payments"));
```
The output json is:
```
{
   "type":"array",
   "items":{"$ref":"#/definitions/Payment"},
   "xml":{"name":"payments","wrapped":true}
}
```

**Comment by frantuma** [comment]
implemented in #2684

**Comment by rbsilva** [comment]
Thanks :)
